.dockerignore

```diff
@@ -0,0 +1,12 @@
+# no need to store it after build
+node_modules
+
+# remove all readme files
+*.md
+
+# no need tem files
+
+*/temp*
```

Comment:
I suggest to add `git` related files to this dockerignore for example .git .gitignore all linting files like eslint

Reply:
also test folder can also been excluded